support/db/user.toml

```diff
@@ -1,3 +1,3 @@
-listen_addresses = "127.0.0.1"
-search_path = "public"
-hba_file = "/hab/svc/postgresql/pg_hba.conf"
```

Comment:
I'm pretty sure this is just for testing and local use. Also, the contents of the `pg_hba.conf` we were using and the contents of the `pg_hba.conf` template that is in the new version of the `core/postgresql` package are compatible.